jdaviz/configs/default/plugins/line_lists/line_lists.py

```diff
@@ -166,9 +166,7 @@ def _on_viewer_data_changed(self, msg=None):
 
         label = msg.data.label
         try:
-            viewer_data = self.app.get_data_from_viewer(
-                self._default_spectrum_viewer_reference_name
-            ).get(label)
+            viewer_data = self.app._jdaviz_helper.get_data(data_label=label)
```

Comment:
Ahh, I understand your desire to make `get_data` available via the app. Personally I think how it is here, while clunky, is probably a more robust way to implement it, but I could be convinced otherwise

Reply:
So is that an approval or a change request? 😅

Reply:
Haha nothing to hold out on. Just a note